### PR TITLE
Update concepts-transactions.md

### PR DESCRIPTION
**Pull request by wongxingjun**
Fix a typo

### CODE REVIEW

File: site2/docs/concepts-transactions.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ All the transaction metadata is persisted in the transaction log. The transactio
...
+Messages produced within a transaction are stored in the transaction buffer. The messages in transaction buffer are not materialized (visible) to consumers until the transactions are committed. The messages in the transaction buffer are discarded when the transactions are aborted. 
 
 ## Pending acknowledge state
 Message acknowledges within a transaction are maintained by the pending acknowledge state before the transaction completes. If a message is in the pending acknowledge state, the message cannot be acknowledged by other transactions until the message is removed from the pending acknowledge state.
 
-The pending acknowledge state is persisted to the pending acknowledge log. The pending acknowledge log is backed by a Pulsar Topic. A new broker can restore the state from the pending acknowledge log to ensure the acknowledgement is not lost.
\ No newline at end of file
+The pending acknowledge state is persisted to the pending acknowledge log. The pending acknowledge log is backed by a Pulsar Topic. A new broker can restore the state from the pending acknowledge log to ensure the acknowledgement is not lost.
```
Comment (Anonymitaet) [inline site2/docs/concepts-transactions.md:29]:
```suggestion
The pending acknowledge state is persisted to the pending acknowledge log. The pending acknowledge log is backed by a Pulsar topic. A new broker can restore the state from the pending acknowledge log to ensure the acknowledgement is not lost.
```

**Review by Anonymitaet — COMMENTED** [review]
Thanks for your contribution. Does this affect only master or other versioned docs? 
If latter, could you please help update all affected versions? Thanks

File: site2/docs/concepts-transactions.md (hunk 1 above)
Comment (wongxingjun) [reply site2/docs/concepts-transactions.md]: Wow, this is another typo.

**Comment by wongxingjun** [comment]
Please check this patch.

**Review by Jennifer88huang — APPROVED** [review]

**Comment by Anonymitaet** [comment]
> Thanks for your contribution. Does this affect only master or other versioned docs?
> If latter, could you please help update all affected versions? Thanks

@wongxingjun any feedback on this? Thanks

**Review by sijie — APPROVED** [review]

**Comment by Anonymitaet** [comment]
@wongxingjun any feedback on this? Thanks

**Comment by wongxingjun** [comment]
> @wongxingjun any feedback on this? Thanks

Same problem exists in both branch 2.7 and 2.8 and I fixed it in 2 new pull requests.

**Comment by Anonymitaet** [comment]
@wongxingjun all your PRs submit same doc changes for the same doc version, please update versioned docs here (https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs) in `master` branch, thanks.

image

image

**Comment by wongxingjun** [comment]
> @wongxingjun all your PRs submit same doc changes for the same doc version, please update versioned docs here (https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs) in `master` branch, thanks.
> 
> image
> 
> image

Done in  #11676

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**